web/assets/main.scss

```
@@ -38,6 +38,21 @@ $govuk-assets-path: "/lpa/frontend/assets/";
   border-bottom: none;
 }
 
+.app-\!-caseworker-summary {
```

These new classes don't need `!-` prefix: that should only be used for override classes (the rest of this file has set a bad example because it's mostly wrong too)
```suggestion
.app-caseworker-summary {
```